lib/ansible/modules/remote_management/oneview/oneview_datacenter.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_datacenter
+short_description: Manage OneView Data Center resources
+description:
+    - "Provides an interface to manage Data Center resources. Can add, update, and remove."
```

Comment:
From the HPE OneView resource documentation:
```
Datacenters
The datacenters (...)
```

Since it is listed as `Datacenters` there, it should likely be refered to as Datacenters and not Data Centers.